src/modules/experience/entities/experience.entity.ts

```diff
@@ -0,0 +1,53 @@
+import { Freelancer } from 'src/modules/freelancer/entities/freelancer.entity';
```

Comment:
same for experience why we need separate module its always updated as part of user edit, I wrote it twice, and still see experience and education modules